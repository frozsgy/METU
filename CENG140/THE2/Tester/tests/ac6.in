ADD a
ADD b
ADD c
ADD d
ADD e
ADD f
SUBSCRIBE a 11
SUBSCRIBE b 22
SUBSCRIBE c 22
SUBSCRIBE d 33
SUBSCRIBE e 33
SUBSCRIBE f 33
SEND a 11
SEND b 22
SEND c 22
SEND d 33
SEND e 33
SEND f 33
UNSUBSCRIBE a 11
UNSUBSCRIBE b 22
UNSUBSCRIBE c 22
UNSUBSCRIBE d 33
UNSUBSCRIBE e 33
UNSUBSCRIBE f 33
SEND a 33
SEND a 22
