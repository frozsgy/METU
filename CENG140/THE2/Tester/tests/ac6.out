1: 7 8
2:
3:
4:
5:
6:
